api/sbom.proto

```diff
@@ -317,30 +317,30 @@ enum SoftwareIdentifierType {
 // It categorizes the roles that software components can fulfill.
 enum Purpose {
     UNKNOWN_PURPOSE = 0; // Unknown purpose.
-    APPLICATION = 1; // Application purpose. (SPDX2.3, CDX1.5, SPDX3.0)
+    APPLICATION = 1; // Application purpose. (SPDX2.3, CDX1.6, SPDX3.0)
```

Comment:
Thanks @idunbarh this is wonderful. It LGTM, I just have one small nit. Can we drop these bumps in the comments here?

```suggestion
    APPLICATION = 1; // Application purpose. (SPDX2.3, CDX1.5, SPDX3.0)
```

Leaving these in the original version where we first matched them allows us to track compatibility by understanding when we first mapped the property and avoids the diff in the go library.